Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:12.13.0
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+#COPY package*.json ./
+
+RUN npm install
```

Comment:
Because docker caches each line specified in the Dockerfile, you should copy package and package-lock.json first, followed by the command to install the dependencies before copying the rest of the files.

This way, the dependencies will only be installed again when there is a change in package.json and package-lock.json.